### PR TITLE
 Show receiver type in the feedback submission form #4451 

### LINKED ISSUE #4451: Show receiver type in the feedback submission form
It says "To:14" here. Not sure what it should be... Came across this while giving feedback for one of my modules.

screen shot 2015-10-09 at 2 55 15 am

on OS X 10.10.5, Chrome 45.

### PR DESCRIPTION
**Pull request by tarun14110**
Fixes #4451

screenshot from 2016-07-05 00 53 19

### CODE REVIEW

File: src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java [hunk 1]
```diff
@@ -33,6 +34,8 @@ public FeedbackSubmissionEditQuestion(FeedbackQuestionAttributes questionAttribu
         this.isModeratedQuestion = isModeratedQuestion;
         isRecipientNameHidden = questionAttributes.isRecipientNameHidden();
         isTeamQuestion = questionAttributes.giverType.equals(FeedbackParticipantType.TEAMS);
```
Comment (kanghj) [inline src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java]: Slightly out of the scope of this PR, but now that `isRecipientTeam` is added, this field should be renamed to `isGiverTeam` to reduce its ambiguity now.

File: src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java [hunk 2]
```diff
@@ -33,6 +34,8 @@ public FeedbackSubmissionEditQuestion(FeedbackQuestionAttributes questionAttribu
...
+        isRecipientTeam = questionAttributes.recipientType.equals(FeedbackParticipantType.TEAMS)
```
Comment (kanghj) [inline src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java]: `questionAttributes.recipientType.isTeam()` should work here

**Comment by kanghj** [comment]
@tarun14110, remember to ping if ready for review. I'll assume that this is ready for review now

**Comment by kanghj** [comment]
@tarun14110, I'll merge this soon. Can you update this branch with master and comment here when done?

**Comment by tarun14110** [comment]
@kanghj updated.

**Pull request merged**